backend/app/views/spree/admin/orders/_adjustments.html.erb

```diff
@@ -1,21 +1,19 @@
-<% if adjustments.eligible.exists? %>
-  <fieldset class="no-border-bottom">
-    <legend><%= title %></legend>
-    <table>
-      <thead>
-        <tr>
-          <th><%= Spree::Adjustment.human_attribute_name(:name)%></th>
-          <th><%= Spree::Adjustment.human_attribute_name(:amount)%></th>
+<fieldset class="no-border-bottom">
+  <legend><%= title %></legend>
+  <table>
+    <thead>
+      <tr>
+        <th><%= Spree::Adjustment.human_attribute_name(:name)%></th>
+        <th><%= Spree::Adjustment.human_attribute_name(:amount)%></th>
+      </tr>
+    </thead>
+    <tbody data-hook="order_details_adjustments"  class="with-border">
+      <% adjustments.eligible.group_by(&:label).each do |label, adjustments| %>
+        <tr class="total">
+          <td><strong><%= label %>:</strong></td>
+          <td class="total align-center"><span><%= Spree::Money.new(adjustments.sum(&:amount), currency: adjustments.first.order.try(:currency)) %></span></td>
```

Comment:
Not related to these changes, but you could add this fix in:
```rb
currency: adjustments.first.order.try(:currency)
```
=>
```rb
currency: order.currency
```

...since you're always passing `order` variable and it's always defined.